src/greentea/gtea/greentea_dlm.py

```diff
@@ -2,28 +2,33 @@
 # Copyright (c) 2021 Arm Limited and Contributors. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 #
-
+"""Implementation of greentea locking. Note: relies on deprecated package lockfile."""
```

Comment:
What's the value of all this lockfile stuff? Why do we need it? This docstring would be the place to explain why the module exists at all.

Reply:
I've added a commit to remove this locking implementation, as it doesn't seem to be used except for registering each greentea instance in "kettle.json".